tests/scripts/generate_psa_tests.py

```diff
@@ -486,6 +504,121 @@ def all_keys_for_usage_flags(self) -> List[StorageKey]:
         self.key_builder = prev_builder
         return keys
 
+    def keys_for_usage_extension(
+            self,
+            extendable: psa_storage.Expr,
+            alg: str,
+            key_type: str,
+            params: Optional[Iterable[str]] = None
+    ) -> List[StorageKey]:
+        """Generate test keys for the specified extendable usage flag,
+           algorithm and key type combination.
+        """
+        keys = [] #type: List[StorageKey]
+        kt = crypto_knowledge.KeyType(key_type, params)
+        for bits in kt.sizes_to_test():
```

Comment:
Minor: here we're going through key types and algorithms, but the key size isn't relevant, so just emitting one test with `bits = kt.sizes_to_test()[0]` would be enough.